lib/core/json_serializer.ex

```diff
@@ -9,18 +9,18 @@ defmodule Polyn.Serializers.JSON do
   @doc """
   Convert a JSON payload into a Polyn.Event struct
   """
-  @spec deserialize(json :: binary(), conn :: Gnat.t()) ::
+  @spec deserialize(json :: binary()) ::
           {:ok, Polyn.Event.t()} | {:error, binary()}
-  def deserialize(json, conn, opts \\ []) do
+  def deserialize(json, opts \\ []) do
```

Comment:
The the schemas are expected to be held locally in a GenServer now, so we don't need a `conn` in these functions to access the Nats server for each message.